include/vcpkg/base/graphs.h

```diff
@@ -2,14 +2,17 @@
 
 #include <vcpkg/base/checks.h>
 #include <vcpkg/base/lineinfo.h>
-#include <vcpkg/base/system.print.h>
+#include <vcpkg/base/messages.h>
 
 #include <string>
 #include <unordered_map>
 #include <vector>
 
 namespace vcpkg::Graphs
 {
+    DECLARE_MESSAGE(GraphCycleDetected, (msg::value), "", "Cycle detected within graph at {value}:");
+    DECLARE_MESSAGE(GraphCycleDetectedElement, (msg::value), "{LOCKED}", "    {value}");
```

Comment:
Hmmm what is "LOCKED"?

I would argue this is locale-invariant?

Reply:
yeah, `{LOCKED}` means "locale-invariant"

Reply:
I mean, it's fine but I'd rather not

Reply:
In contrast I rather would avoid complicating our localization with trivial things like "add 4 spaces in front of this string". I think `unsafe_blah` is better and more declarative for this purpose.